mobile/lib/pages/common/settings.page.dart

```diff
@@ -8,36 +8,69 @@ import 'package:immich_mobile/widgets/settings/asset_list_settings/asset_list_se
 import 'package:immich_mobile/widgets/settings/asset_viewer_settings/asset_viewer_settings.dart';
 import 'package:immich_mobile/widgets/settings/backup_settings/backup_settings.dart';
 import 'package:immich_mobile/widgets/settings/language_settings.dart';
+import 'package:immich_mobile/widgets/settings/networking_settings/networking_settings.dart';
 import 'package:immich_mobile/widgets/settings/notification_setting.dart';
 import 'package:immich_mobile/widgets/settings/preference_settings/preference_setting.dart';
 import 'package:immich_mobile/routing/router.dart';
 
 enum SettingSection {
```

Comment:
I was initially thinking about shortening the subtitles, but now I'm looking at it more closely and the subtitles are all just restating the title. There's no new info. The sections are all self-explanatory, so I think you can just get rid of the subtitles. It'll look cleaner.

Reply:
Also, some of the entries are thicker than others because the subtitle is two lines vs. one, so removing them will make the thickness consistent.

Reply:
It doesn't look good without subtitles. I like having subtitles in this UI element as a design preference